packages/recommendations-js/README.md

```diff
@@ -1,13 +1,13 @@
-# `@algolia/js-recommendations`
+# `@algolia/recommendations-js`
 
 JavaScript package for [Algolia Recommend](https://www.algolia.com/doc/guides/algolia-ai/recommend/).
 
 ## Installation
 
 ```sh
-yarn add @algolia/js-recommendations@experimental
+yarn add @algolia/recommendations-js@experimental
 # or
-npm install @algolia/js-recommendations@experimental
+npm install @algolia/recommendations-js@experimental
```

Comment:
we'll need to make sure to `npm deprecate` the older package names once this is merged

Reply:
I added that point in the "Next steps" section—thanks!